dashboard/app/views/home/_homepage.html.haml

```diff
@@ -14,6 +14,13 @@
 
 .clear{style: "clear: both"}
 
+- if current_user
```

Comment:
Should we mark a container `display: none` so that there's no risk of this displaying outside of the React copy?